apps/parktime/park_time.star

```diff
@@ -0,0 +1,339 @@
+    {"name": "Universal Studios", "id": "9fc68f1c-3f5e-4f09-89f2-aab2cf1a0741", "entityType": "DESTINATION", "color": "#FFFFFF"},
+    {"name": "Universal Orlando Resort", "id": "89db5d43-c434-4097-b71f-f6869f495a22", "entityType": "DESTINATION", "color": "#FFFFFF"},
+    {"name": "Disneyland Park", "id": "7340550b-c14d-4def-80bb-acdb51d49a66", "entityType": "PARK", "color": "#573eaf"},
+    {"name": "Disney California Adventure Park", "id": "832fcd51-ea19-4e77-85c7-75d5843b127c", "entityType": "PARK", "color": "#8260c2"},
+    {"name": "Magic Kingdom Park", "id": "75ea578a-adc8-4116-a54d-dccb60765ef9", "entityType": "PARK", "color": "#7236ba"},
+    {"name": "EPCOT", "id": "47f90d2c-e191-4239-a466-5892ef59a88b", "entityType": "PARK", "color": "#ce019e"},
+    {"name": "Disney's Hollywood Studios", "id": "288747d1-8b4f-4a64-867e-ea7c9b27bad8", "entityType": "PARK", "color": "#e90129"},
+    {"name": "Disney's Animal Kingdom Theme Park", "id": "1c84a229-8862-4648-9c71-378ddd2c7693", "entityType": "PARK", "color": "#71a750"},
+    {"name": "Disney's Typhoon Lagoon Water Park", "id": "b070cbc5-feaa-4b87-a8c1-f94cca037a18", "entityType": "PARK", "color": "#408297"},
+    {"name": "Disney's Blizzard Beach Water Park", "id": "ead53ea5-22e5-4095-9a83-8c29300d7c63", "entityType": "PARK", "color": "#31b0f0"},
+    {"name": "Universal's Volcano Bay", "id": "fe78a026-b91b-470c-b906-9d2266b692da", "entityType": "PARK", "color": "#0468d9"},
+    {"name": "Universal Studios Florida", "id": "eb3f4560-2383-4a36-9152-6b3e5ed6bc57", "entityType": "PARK", "color": "#0468d9"},
+    {"name": "Universal's Islands of Adventure", "id": "267615cc-8943-4c2a-ae2c-5da728ca591f", "entityType": "PARK", "color": "#0468d9"},
+    {"name": "Universal Studios", "id": "bc4005c5-8c7e-41d7-b349-cdddf1796427", "entityType": "PARK", "color": "#0468d9"},
+]
+
+# Format the Schema Options
+def entity_to_option(entity):
+    if entity["entityType"] == "PARK":
+        return schema.Option(
+            display = "PARK HOURS - " + entity["name"],
+            value = entity["id"],
+        )
+    elif entity["entityType"] == "SHOW":
+        return schema.Option(
+            display = "SHOW TIMES - " + entity["name"],
+            value = entity["id"],
+        )
+    elif entity["entityType"] == "ATTRACTION":
+        return schema.Option(
+            display = "WAIT TIME - " + entity["name"],
+            value = entity["id"],
+        )
+    else:
+        return schema.Option(
+            display = entity["name"],
+            value = entity["id"],
+        )
+
+# Get the Schema Options for Destinations
+def get_destination_options():
+    return [entity_to_option(destination) for destination in MAIN_ENTITIES if destination["entityType"] == "DESTINATION"]
+
+# Get the Schema Options for Entities
+def get_entity_options(park):
+    res = http.get(THEME_PARKS_WIKI_URL + park)
```

Comment:
This result also doesn't seem to be used. Is it to verify that the configuration option is correct? If so, I'd assume, the subsequent calls would fail?

Reply:
Rednudancy removed.